lib/awskeyring/awsapi.rb

```diff
@@ -47,6 +60,12 @@ def self.get_token(params = {}) # rubocop:disable  Metrics/AbcSize, Metrics/Meth
               serial_number: params[:mfa],
               token_code: params[:code]
             )
+          else
+            sts.get_federation_token(
+              name: params[:user],
```

Comment:
Name can only be up to 32 characters. This will bomb for anyone with usernames longer than that